src/tests/setup.ts

```diff
@@ -0,0 +1,9 @@
+import { afterEach } from "vitest";
+import { cleanup } from "@testing-library/react";
+import "@testing-library/jest-dom/vitest";
```

Comment:
That's funny

Reply:
lol 😆 yeah, testing-library had their code too coupled to "jest" implementation